AzureTable/Microsoft.DataTransfer.AzureTable/Source/AzureTableSourceAdapter.cs

```diff
@@ -31,9 +32,14 @@ public AzureTableSourceAdapter(IAzureTableSourceAdapterInstanceConfiguration con
 
             string connectionString = System.Text.RegularExpressions.Regex.Replace(
                 configuration.ConnectionString, @"(TableEndpoint=https://)(.*\.)(documents)(\.azure\.com)",
-                m => m.Groups[1].Value + m.Groups[2].Value + "table.cosmosdb" + m.Groups[4].Value).TrimEnd('/');
+                m => m.Groups[1].Value + m.Groups[2].Value + "table.cosmosdb" + m.Groups[4].Value);
 
-            var client = CloudStorageAccount.Parse(connectionString).CreateCloudTableClient();
+            TableConnectionPolicy connectionPolicy = new TableConnectionPolicy()
```

Comment:
Shouldn't all this be configurable, like for SQL API in Cosmos DB?

Reply:
this is only a table to log failures. I don't think all of these need to be exposed, so exposing only the connection string for now if the user wishes to log in a specific account.